### ISSUE TITLE
Foundation.MediaQuery.current not working

### ISSUE THREAD

**Issue by alexryans** [zurb/foundation-sites#10412]
#### How to reproduce this bug:

1. Print out the current media query breakpoint using `console.log('Current Breakpoint: ', Foundation.MediaQuery.current);`

#### What should happen:
The JavaScript Console should display `Current Breakpoint: small` (or otherwise, depending on which breakpoint is currently active).

#### What happened instead:
The JavaScript Console outputs `Current Breakpoint: `. No errors are thrown, pertaining to this or any other issues.

<img width="561" alt="screen shot 2017-07-14 at 13 22 43" src="https://user-images.githubusercontent.com/3007053/28212040-94923a1a-6897-11e7-9de7-d1bc9f382958.png">

#### Browser(s) and Device(s) tested on:
Chrome 59.0.3071.115

#### Foundation Version(s) you are using:
6.4.1

#### Other information
All other included Foundation modules and JavaScript is working correctly (e.g. accordions etc.), it's just the actual Media Query check (and anything that acts on the results of said check) that isn't running.

**Comment by kball** [comment]
This is a duplicate of https://github.com/zurb/foundation-sites/issues/10363

There's a fix in that will be in the next point release (prob early next week), in the meantime the work around is to call `Foundation.MediaQuery._init()` after including Foundation

**Comment by alexryans** [comment]
Great, thanks!

**Comment by kball** [comment]
closing based on #10369, the fix will be in 6.4.2 currently targeted for next Monday release.

**Comment by RvWensen** [comment]
```
if (Foundation.MediaQuery.is('small')) {
  // True if small
  // False if medium or large
}
```
its not working
CHECK: https://codepen.io/rvwensen/pen/QoVRKx?editors=1111
